dcos/marathon.py

```diff
@@ -56,6 +56,10 @@ def _to_exception(response):
         return DCOSException(
             'Error while fetching [{0}]: HTTP {1}: {2}'.format(
                 response.request.url, response.status_code, response.reason))
+    elif response.status_code == 409:
+        return DCOSException(
+            'App is locked by one or more deployments. '
```

Comment:
`App` -> `App or group` since this can affect either (and in this case I think the Marathon API error message which refers only to an `App` is confusing).